modules/standard/IO.chpl

```diff
@@ -6583,7 +6583,11 @@ inline proc fileWriter.readWriteNewline() throws
 
 /* Returns `true` if this fileReader is configured for binary I/O.
  */
-proc fileReader.binary():bool {
+@deprecated(notes="'fileReader.binary()' is deprecated; check whether the fileReader is configured with a binary deserializer instead")
```

Comment:
Maybe instead we say:
```
...; please use 'fileReader.deserializerType' to check for a binary deserializer instead
```

Same for the fileWriter case as well, just with ``serializerType``.